test-case/verify-kernel-boot-log.sh

```diff
@@ -46,14 +46,24 @@ main()
 
 wait_is_system_running()
 {
-    local manager="$1"
+    local manager="$1" # --user or --system
+    local wait_secs=30 ret=0
+    local cmd=(systemctl "$manager" --wait is-system-running)
 
-    printf 'systemctl %s --wait is-system-running: ' "$manager"
-    systemctl "$manager" --wait is-system-running || {
-        systemctl "$manager" --no-pager --failed
+    printf '%s\n' "${cmd[*]}"
+    timeout -k $((wait_secs + 5)) "$wait_secs" "${cmd[@]}" || ret=$?
```

Comment:
Why wait_secs + 5?

Reply:
`man timeout` ?

Reply:
I also have a question here. In this case, my understanding is that timeout runs the command for `$wait_secs`, and if it is not terminated, it will kill it after 5 seconds. I think `wait_secs + 5` is not correct.

Reply:
`-k` is offset after the initial signal is sent, so `wait_secs + 5` seems to double the timeout.

Reply:
You're right, it should be `timeout -k 5 "$wait_secs". Good catch thank you, will update.